specs/yara/yara.table

```diff
@@ -9,6 +9,8 @@ schema([
         additional=True),
     Column("sigfile", TEXT, "Signature file used",
         additional=True),
+	Column("sigrules", TEXT, "Signature strings used",
```

Comment:
Is this a single rule, or a collection or rules? If they latter, how are the concatenated?

Reply:
You can pass multiple rules as well with queries running with each rule. They are not concatenated. 

```
select * from yara where path like '%' and sigrules in ('rule foo: bar {strings: $a = "lmn" condition: $a}', 'rule always_false { condition: false }', 'rule always_true { condition: true }');
```

Reply:
that's a single rule per constraint, and thus a single rule per row. I think singular is correct here.